Nos/Router.swift

```diff
@@ -58,7 +58,11 @@ import Dependencies
         if let identifier = note.identifier {
             push(.note(.identifier(identifier)))
         } else if let replaceableIdentifier = note.replaceableIdentifier, let author = note.author {
-            push(.note(.replaceableIdentifier(replaceableID: replaceableIdentifier, author: author)))
+            push(
+                .note(
+                    .replaceableIdentifier(replaceableID: replaceableIdentifier, author: author, kind: note.kind)
```

Comment:
It occurs to me now that `replaceableIdentifier` makes it sound like an identifier that could be replaced. But really the identifier is staying the same and the rest of the event gets replaced... not sure I want to push a rename atm but maybe `replaceableEventIdentifier`, or `persistentIdentifier`... or `platonicIdentifier` (lol) would make more sense.

Reply:
I think I like `replaceableEventIdentifier` for the future. I might even consider it for now.